charts/openshift-metering/values.yaml

```diff
@@ -458,6 +463,10 @@ presto:
       metastore:
         affinity: {}
         config:
+          tls:
+            enabled: false
```

Comment:
We will need a follow-up PR that adds the `certificate`, `key`, and `caCertificate` fields so the user can specify those directly.